sync/handlers.py

```diff
@@ -55,14 +55,16 @@ def handle_pr(git_gecko, git_wpt, event):
 
         merge_sha = (event["pull_request"]["merge_commit_sha"]
                      if event["pull_request"]["merged"] else None)
+        merged_by = (event["pull_request"]["merged_by"]["login"] if merge_sha else None)
         with SyncLock.for_process(sync.process_name) as lock:
             with sync.as_mut(lock):
                 update_func(git_gecko,
                             git_wpt,
                             sync,
                             event["action"],
                             merge_sha,
-                            event["pull_request"]["base"]["sha"])
+                            event["pull_request"]["base"]["sha"],
+                            merged_by)
```

Comment:
This can end up calling `downstream.update_pr` with an extra argument.

Reply:
I have added a kwarg in the downstream.update_pr [here](https://github.com/mozilla/wpt-sync/pull/439/files#diff-9145f0a692affb6ee2e22ccd4014b221R1050) to avoid this.